website/docs/indexing.md

```diff
@@ -27,27 +27,19 @@ could end up having to merge all the base files against all incoming updates/del
 Currently, Hudi supports the following index types. Default is SIMPLE on Spark engine, and INMEMORY on Flink and Java 
 engines.
 
-- **BLOOM:** Employs bloom filters built out of the record keys, optionally also pruning candidate files using 
-  record key ranges.Key uniqueness is enforced inside partitions.
-- **GLOBAL_BLOOM:** Employs bloom filters built out of the record keys, optionally also pruning candidate files using 
-  record key ranges. Key uniqueness is enforced across all partitions in the table.
-- **SIMPLE (default for Spark engines):** Default index type for spark engine. Performs a lean join of the incoming records against keys extracted from the table on 
-  storage. Key uniqueness is enforced inside partitions. 
+- **BLOOM:** Uses bloom filters generated from record keys, with the option to further narrow down candidate files based on the ranges of the record keys. This method ensures the uniqueness of keys within partitions.
+- **GLOBAL_BLOOM:** Utilizes bloom filters created from record keys, and may also refine the selection of candidate files by using the ranges of record keys. This ensures that keys are unique throughout all partitions in the table.
+- **SIMPLE (default for Spark engines):** This is the standard index type for the Spark engine. It executes an efficient join of incoming records with keys retrieved from the table stored on disk, ensuring that keys are unique within partitions. 
 - **GLOBAL_SIMPLE:** Performs a lean join of the incoming records against keys extracted from the table on
   storage. Key uniqueness is enforced across all partitions in the table.
```

Comment:
there is a misconception: index does not ensure key uniqueness, it requires keys to be partition-level or table/global-level unique so it can function correctly. we should phrase it that way.

Reply:
thanks @xushiyan will re-word.

Reply:
@xushiyan are you referring to global_simple description?